### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by paramaggarwal** [tgds/atom-touchbar#2]
This is actually the best touchbar package that I came across for Atom Editor. It covers the most common use cases.

1. I just wish that the pallette button wasn't that bright yellow colour. As well as the linter button was not that bright blue colour. The icons on touchbar are usually white icons on grey to blend in with the keyboard. (Opened #3)
2. The buttons could look better by using a SegmentedControl. (See https://github.com/paramaggarwal/atom-touchbar/blob/master/lib/touchbar-buttons.js)

Thank you for the package!

**Comment by tgds** [comment]
Hi Param, thank you very much for your suggestions. I've almost forgotten about this package, but maybe it's time to get back to it. 

I've reworked the styling of the buttons and removed the colours, so it blends much better. 
Thanks for the suggestion on SegmentedControl. I cannot yet use it for the palette and search. I've set custom grey icons for these two buttons and it seems that there's a bug with SegmentedControls, where these buttons don't scale custom icons correctly, so I used standard buttons with small spacing instead. I used SegmentedControl for the Linter section though.
